shopfloor_batch_automatic_creation/actions/picking_batch_auto_create.py

```diff
@@ -110,7 +113,7 @@ def _search_pickings(self, picking_types, user=None, commercial_partner=None):
     def _sort(self, pickings):
         return pickings.sorted(
             lambda picking: (
-                -(int(picking.priority) if picking.priority else 1),
+                -(int(picking.priority) if picking.priority else PRIORITY_NORMAL),
```

Comment:
why this change ? `PROCUREMENT_PRIORITIES[0][0]` is "0"

Reply:
Thanks @mmequignon for your comment. The value 1 used to be `Normal`. After below Odoo's change, the `Normal` is now 0.
https://github.com/odoo/odoo/commit/e40553723b893bcd2244e9b9fcc5c1a7e1bf27d3#diff-55c6314416a6a400da6acd5018d161a55eeeb0e3008fec8828121e3dd12be0ebL19-R19